src/librustc/middle/expr_use_visitor.rs

```diff
@@ -2,25 +2,20 @@
 //! normal visitor, which just walks the entire body in one shot, the
```

Comment:
can you check whether clippy still works after your PR? We use this visitor a lot.

Reply:
Oh... anything using EUV should've transitioned a long time ago to being based on MIR, IMO. I wish I brought this up to clippy people earlier :/